public/assets/images/icons/headerIcons/close.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
+import { IconProps } from '../../../../../src/features/common/types/common';
 
-function Close({ color }: any) {
+function Close({ color }: IconProps) {
```

Comment:
Set up default color while destructuring the props

Reply:
It is easier to specify the default color here instead of adding conditional statements in the code

`function Close({ color = '#1e2225' }: IconProps) `